api/src/main/java/org/openmrs/module/fhir/api/impl/ObsServiceImpl.java

```diff
@@ -223,5 +226,23 @@ public void deleteObs(String id) {
 		Obs obs = Context.getObsService().getObsByUuid(id);
 		Context.getObsService().voidObs(obs, "DELETED by FHIR Request");
 	}
+	
+	/**
+	 * @see org.openmrs.module.fhir.api.ObsService#createFHIRObservation(Observation)
+	 */
+	@Override
+	public Observation createFHIRObservation(Observation observation){		
+		List<String> errors = new ArrayList<String>();
+		Obs obs=FHIRObsUtil.generateOpenMRSObs(observation,errors);
```

Comment:
@surangak  @harsha89  Oops I have edited that after running the formatter. My formatter doesn't run when I build the code. So I set it to run when I save the code every time. Java --> Editor --> Save Actions

Here after this mistakes will not happen ;)